src/modules/user/userSwagerDocs.js

```diff
@@ -1,478 +1,499 @@
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         required: true
+ *         description: String ID of the user to retrieve.
+ *         schema:
+ *           type: String
+ *     responses:
+ *       200:
+ *         description: A single user.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 data:
+ *                   type: object
+ *                   properties:
+ *                     id:
+ *                       type: String
+ *                       description: The user ID.
+ *                       example: fcadcac5-b894-4150-818c-7d6f0e730c59
+ *                     firstName:
+ *                       type: string
+ *                       description: The user's first name.
+ *                       example: John
+ *                     lastName:
+ *                       type: string
+ *                       description: The user's last name.
+ *                       example: Smith
+ *                     email:
+ *                       type: string
+ *                       format: email
+ *                       description: The user's email.
+ *                       example: example@example.com
+ *                     avatar:
+ *                       type: string
+ *                       format: URL
+ *                       description: Avatar URL.
+ *                       example: example.com/example.jpeg
+ *                     about:
+ *                       type: string
+ *                       description: Details about te user.
+ *                       example: "I'm a student of front-end....."
+ *                     skills:
+ *                       type: [string]
+ *                       description: Array of users skills.
+ *                       example: [HTML, CSS, JavaScript, Git, GitHub]
+ *                     applications:
+ *                       type: [object]
+ *                       description: An array of application objects
+ *                       example: [application]
+ *                     offers:
+ *                       type: [object]
+ *                       description: An array of offers objects
+ *                       example: [offer]
+ *                     listings:
+ *                       type: [object]
+ *                       description: An array of listing objects
+ *                       example: [listing]
+ *                     Created:
+ *                       type: string
+ *                       format: date
+ *                       description: Date user joined.
+ *                       example: 2023-01-16T19:55:13.609Z
+ *                     Updated:
+ *                       type: string
+ *                       format: date
+ *                       description: Most recent update.
+ *                       example: 2023-01-16T19:55:13.609Z
+ *       500:
+ *         description: Internal server error

```

Comment:
Once these changes get merged we will have more errors on put
https://github.com/NoroffFEU/agency-api.noroff.dev/blob/Users-Testing-Errors/src/modules/user/controllers/controllerUpdate.js

Reply:
done